index.html

```diff
@@ -0,0 +1,695 @@
+                                            </g>
+                                        </svg>
+                                        <span>[number]</span>
+                                    </div>
+                                </a>
+                                <a class="wrap share" href="./postshare.html">
+                                    <div>
+                                        <svg viewBox="0 0 24 24" aria-hidden="true" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px">
+                                            <g>
+                                            <path d="M12 2.59l5.7 5.7-1.41 1.42L13 6.41V16h-2V6.41l-3.3 3.3-1.41-1.42L12 2.59zM21 15l-.02 3.51c0 1.38-1.12 2.49-2.5 2.49H5.5C4.11 21 3 19.88 3 18.5V15h2v3.5c0 .28.22.5.5.5h12.98c.28 0 .5-.22.5-.5L19 15h2z"></path>
+                                            </g>
+                                        </svg>
+                                    </div>
+                                </a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="right">
+                <div class="search">
+                    <div class="searchfield">
+                        <svg viewBox="0 0 24 24" aria-hidden="true">
+                            <g>
+                                <path d="M10.25 3.75c-3.59 0-6.5 2.91-6.5 6.5s2.91 6.5 6.5 6.5c1.795 0 3.419-.726 4.596-1.904 1.178-1.177 1.904-2.801 1.904-4.596 0-3.59-2.91-6.5-6.5-6.5zm-8.5 6.5c0-4.694 3.806-8.5 8.5-8.5s8.5 3.806 8.5 8.5c0 1.986-.682 3.815-1.824 5.262l4.781 4.781-1.414 1.414-4.781-4.781c-1.447 1.142-3.276 1.824-5.262 1.824-4.694 0-8.5-3.806-8.5-8.5z"></path>
+                            </g>
+                        </svg>
+                        <input type="text" placeholder="Search Mint">
+                    </div>
+                </div>
+                <div class="trendingarea">
+                    <h1>Trends for you</h1>
+                    <div class="trendingtopic">
+                        <aside>
+                            <a href="./trendingoptions.html">
+                                <svg viewBox="0 0 24 24" aria-hidden="true">
+                                    <g>
+                                        <path d="M3 12c0-1.1.9-2 2-2s2 .9 2 2-.9 2-2 2-2-.9-2-2zm9 2c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm7 0c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2z"></path>
+                                    </g>
+                                </svg>
+                            </a>
+                        </aside>
+                        <a href="./trendingtopic.html">
+                            <p>{trending in}</p>
```

Comment:
I was guessing there was a way to link HTML files. Good use here!

Reply:
The parts in the curly brackets are just placeholder text because I wasn't creative enough to come up with anything